Mathlib/Data/List/Defs.lean

```diff
@@ -488,9 +488,12 @@ theorem length_mapAccumr₂ :
 
 end MapAccumr
 
+/- #adaptation_note: this attribute should be removed after Mathlib moves to v4.15.0-rc1. -/
+set_option allowUnsafeReducibility true in
+attribute [semireducible] Fin.foldr.loop
```

Comment:
Does this apply globally, or just for the current file?

Reply:
I also worry that this PR might have accumulated a bunch of proof uglification before we discovered this fix; is that the case, or do I have the chronology wrong?

Reply:
The uglification seems to still be necessary at the moment, and I'm not really sure why it is happening. It is only at the level of having to redo some simp only blocks, however.

Reply:
> Does this apply globally, or just for the current file?

This is global.